is/utils/io.hpp

```diff
@@ -1,10 +1,64 @@
 #pragma once
 
 #include <string>
+#include <fstream>
 #include "google/protobuf/message.h"
 #include "status.hpp"
+#include "boost/optional.hpp"
+
+namespace google {
+namespace protobuf {
+namespace io {
+class ZeroCopyOutputStream;
+class CodedOutputStream;
+class CodedInputStream;
+}
+}
+}
 
 namespace is {
 wire::Status load(std::string const& filename, google::protobuf::Message* message);
 wire::Status save(std::string const& filename, google::protobuf::Message const& message);
+
+struct ProtobufWriter {
+  std::ofstream file;
+  google::protobuf::io::ZeroCopyOutputStream *raw_output;
```

Comment:
Use OstreamOutputStream instead of the interface to avoid unnecessary dynamic allocation?

Reply:
I tried to use smart pointers but it seems that the buffer weren't flush (read the generated file with a python script). I'm going to post this issue on this [stackoverflow](https://stackoverflow.com/questions/2340730/are-there-c-equivalents-for-the-protocol-buffers-delimited-i-o-functions-in-ja) question that I've based for this last commit.